AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/src/KeyWrapping/EdkWrapping.dfy

```diff
@@ -168,6 +176,8 @@ module EdkWrapping {
                  wrapInfo := ret.value.wrapInfo)),
              [])
   {
+
+      // TODO require this
```

Comment:
Why not require it then? Did i leave this?

Reply:
Yes, it's in https://github.com/aws/aws-cryptographic-material-providers-library/pull/909